### PR TITLE
feat: added ujust rebase-secureblue

### LINKED ISSUE #692: [FEAT] ujust rebase-secureblue
**Describe the benefit this feature would provide to the project**
Allow users to more easily rebase to alternative secureblue image's such as kinonite, nvidia-open, (as opposed to nvidia) etc. once they've already installed secureblue.

**Describe the solution you'd like**
ujust script potentially based on `install_secureblue.sh` (or directly calling it) that would streamline rebasing to other secureblue images once they're already inside a secureblue image. 

**Declaration**
- [x] I declare that I have read the README and my feature request is in-scope.

### PR DESCRIPTION
**Pull request by ShadowSlayer1441**
Closes #692

### CODE REVIEW

**Comment by mintpilo** [comment]
It's probally not an issue for now, but after this gets merged, this will technically be duplicate code, might make sense to later delete the old location to `install_secureblue.sh`, and then have actions include the file from this location into releases. (I'm not really sure what the best way to go about this is, or what would be preferred here)

**Comment by ShadowSlayer1441** [comment]
> It's probally not an issue for now, but after this gets merged, this will technically be duplicate code, might make sense to later delete the old location to `install_secureblue.sh`, and then have actions include the file from this location into releases. (I'm not really sure what the best way to go about this is)

I thought about this, the old location doesn't get stored in the OS (not that 88 lines of text really matters at all). We could change the button to refer to a different location, but unfortunately github doesn't support direct download links unless we post it as a github page. The current solution is to directly add the install.sh to every release, which works. I am concerned that we now have two versions of  `install_secureblue.sh`, not sure how to avoid it though.

File: files/justfiles/utilities.just [hunk 1]
```diff
@@ -228,3 +228,7 @@ install-steam:
             systemctl reboot
         fi
     fi
+
+rebase-secureblue:
+    #!/usr/bin/bash
+    sh /usr/share/secureblue/install_secureblue.sh
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just]: (nit) /usr/bin/bash instead of sh

File: files/justfiles/utilities.just [hunk 2]
```diff
@@ -228,3 +228,7 @@ install-steam:
             systemctl reboot
         fi
     fi
+
+rebase-secureblue:
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just:233]: needs a comment. maybe:

```bash
# Rebase between secureblue images
```

**Review by RoyalOughtness — APPROVED** [review]

**Review by RKNF404 — APPROVED** [review]

**Pull request merged**